docs/src/content/docs/guides/components.mdx

```diff
@@ -87,6 +87,48 @@ The code above generates the following tabs on the page:
 	</TabItem>
 </Tabs>
 
+#### Synced tabs
+
+Multiple sets of tabs on the same page can be synced so that switching tabs in one switches the others to a tab with the same label.
```

Comment:
Suggested rewording here. I’m still not super happy — maybe @sarah11918 would have some good ideas for explaining this clearly.
```suggestion
Keep multiple sets of tabs on the same page in sync with the `syncKey` attribute. Switching to a synced tab will also switch to the tab with the same label in other `<Tabs>` that share the same `syncKey`.
```